cmake/OmimOptions.cmake

```diff
@@ -0,0 +1,19 @@
+set(CMAKE_BUILD_TYPE "Release" CACHE STRING "Build type")
```

Comment:
Previous logic set it only if it wasn't provided explicitly from the command line.

Reply:
This logic is similar but it's one-liner
Release by default but you can overwrite it